server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json

```diff
@@ -130,114 +130,116 @@
               {
                 "tiles": [
                   {
-                    "rankingTileSpec": {
-                      "rankingCount": 10,
-                      "showHighest": true
-                    },
                     "statVarKey": [
-                      "Count_Person_Rural_BelowPovertyLevelInThePast12Months"
+                      "sdg/SI_POV_DAY1"
```

Comment:
there was a change in ordering of charts. All original charts still show up & now there is an additional charts from sdg_1.2 topic